sass/style.sass

```diff
@@ -1264,6 +1267,71 @@ textarea.deploy-key
 	p, h1, h2, h3, h4, h5, .h1, .h2, .h3, .h4, .h5, .close
 		color: #fff
 
+.modal.deploy
+
+	.modal-lg
+		@media screen and (min-width: 992px)
+			width: 90%
+			color: #505050
+
+	.modal-dialog
+		margin: 0 auto
+
+		.modal-content
+			border-radius: 0
+			background-color: #F0F3F4
+
+			.modal-header
+				border-radius: 0
+				background: #ffffff
+
+				.modal-title
+					color: #000000
+
+				.close
+					color: #000000
+
+			.modal-body
+				// ensure that there is a scrollbar inside the modal
+				overflow-y: auto
+				height: 100%
+				// we remove padding here so the menu can do a left aligned active indicator
+				padding-left: 0
+
+
+				.menu
+					padding-right: 0
+
+					.step
+						font-size: 16px
+						color: #B2B3B4
```

Comment:
are any of these colours going to be re-used? might be worth doing them as variables if so